src/connectors/statusChecksDb/status-checks.connector.js

```diff
@@ -0,0 +1,242 @@
+"use strict";
+
+const { logEmitter, ERROR } = require("../../services/logging.service");
+const { establishConnectionToCosmos } = require("../cosmos.client");
+const { decryptId } = require("../../utils/crypto");
+
+const findRegistrationByFsaId = async (fsa_rn) => {
```

Comment:
Potential risk if fsa_rn is not validated in router/controller or not catch by WAF. Same apply for the rest of functions in this file.